code/game/objects/items/mail_box.dm

```diff
@@ -0,0 +1,100 @@
+/*
+
+	Mailboxes/P.O. Boxes! (idk which one fits more)
+	Contains mail, using it with an empty hand opens a menu for you to pull out your own mail.
+	Having cargo bay access lets you pull out anyone's mail, so it doesn't get trapped in limbo
+
+	made by candycane/etherware
+
+*/
+
+
+/obj/machinery/mailbox
```

Comment:
why is it a machine? couldn't it just be a structure?

Reply:
oh